### PR TITLE
DSW-2182: initial checkbox-group set up

### PR DESCRIPTION
**Pull request by dandel10n**
Checkbox-group component set up into forms for all the apps

### CODE REVIEW

File: nextjs-app-v13/src/pages/integrations/form.tsx [hunk 1]
```diff
@@ -230,8 +243,29 @@ export default function Form() {
...
                         name="newsletterSignup"
                         checked={newsletterSignup}
-                        onChange={handleNewsletterSignup}
-                    >Receive discounts, loyalty offers and other updates via email</PieCheckbox>
+                        onChange={handleNewsletterSignup}>
+                        Receive discounts, loyalty offers and other updates via email
+                    </PieCheckbox>
+
+                    <PieCheckboxGroup>
+                        <PieFormLabel>Choose the way we can contact you:</PieFormLabel>
```
Comment (xander-marjoram) [inline nextjs-app-v13/src/pages/integrations/form.tsx:251]: I think maybe we need to use `slot="label"` here?

File: nextjs-app-v14/src/app/integrations/form/form.tsx [hunk 2]
```diff
@@ -231,8 +244,28 @@ export default function Form() {
                         data-test-id="newsletterSignup"
                         name="newsletterSignup"
                         checked={newsletterSignup}
-                        onChange={handleNewsletterSignup}
-                    >Receive discounts, loyalty offers and other updates via email</PieCheckbox>
+                        onChange={handleNewsletterSignup}>
+                        Receive discounts, loyalty offers and other updates via email
+                    </PieCheckbox>
+                    <PieCheckboxGroup>
+                        <PieFormLabel>Choose the way we can contact you:</PieFormLabel>
```
Comment (xander-marjoram) [inline nextjs-app-v14/src/app/integrations/form/form.tsx:251]: And here too, `slot="label"`

File: nextjs-app-v14/test/visual/nextjs.spec.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ describe('NextJS Aperture App', () => {
         { url: '/components/tag', name: 'Tag' },
         { url: '/components/text-input', name: 'Text Input' },
         { url: '/components/textarea', name: 'Textarea' },
+        { url: '/components/checkbox', name: 'Checkbox' },
```
Comment (xander-marjoram) [inline nextjs-app-v14/test/visual/nextjs.spec.js:23]: Nit: Can this be added alphabetically please

**Comment by jamieomaguire** [comment]
@leksaBoiko please can we add `checkbox-group` to the SSR test spec?

**Review by xander-marjoram — DISMISSED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**